DifferentialGrowth2D/DifferentialGrowth2D.js

```diff
@@ -0,0 +1,61 @@
+import { Rectangle } from "./rectangle.js";
+import { Quadtree } from "./quadtree.js";
+import { DifferentialPolyline } from "./DifferentialPolyline.js";
+
+const WIDTH = 500;
+const HEIGHT = 700;
+const BOUNDS = new Rectangle(0, 0, WIDTH, HEIGHT);
+const QUADTREE = new Quadtree(BOUNDS);
+
+const points = [];
+const points2 = [];
+const INITIAL_POINTS = 20;
+const OFFSET = HEIGHT / 12;
+for (let i = 0; i < INITIAL_POINTS; i++) {
+  const angle = (2.0 * Math.PI * i) / INITIAL_POINTS;
+  const x = WIDTH / 2 + 50 * Math.cos(angle);
+  const y = HEIGHT / 2 + 50 * Math.sin(angle);
+  points.push([x, y - OFFSET]);
+  points2.push([x, y + OFFSET]);
+}
+
+const POLYLINE = new DifferentialPolyline(points, QUADTREE);
+const POLYLINE2 = new DifferentialPolyline(points2, QUADTREE);
+
+const DELTA_TIME = 0.1;
+
+export const sketch = (p) => {
```

Comment:
Maybe add a button to toggle visibility of the reference geometry (quadtrees, force circles, etc)